packages/frontend/src/components/RuntimeAdapter.tsx

```diff
@@ -52,6 +59,10 @@ export default function RuntimeAdapter({
           await window.__selectAccount(notificationAccountId)
         } else if (chatId !== 0) {
           await selectChat(accountId, chatId)
+          if (isWebxdcInfo) {
+            const message = await BackendRemote.rpc.getMessage(accountId, msgId)
+            internalOpenWebxdc(accountId, message)
+          }
```

Comment:
Is it intentional that we don't open the app if the notification is for a different account?

Reply:
Decided to not open apps on notification click, so this will be resolved.

Reply:
Is obsolete now